main/static/index.html

```diff
@@ -71,9 +71,15 @@
 
     <div>
       <div class="col-xs-12">
-        <table ng-show="isTabular()" class="table table-striped">
+        <table ng-show="isTabular() && queryResult.name != 'describe'" class="table table-striped">
           <tr ng-repeat="row in queryResult.body"><td>{{row}}</td></tr>
         </table>
+        <div ng-show="isTabular() && queryResult.name == 'describe'">
+          <table ng-repeat="(key, list) in queryResult.body" class="table table-striped table-column">
```

Comment:
It would be a bit strange. The columns aren't the same lengths, so you'd have a very, very jagged table in lots of cases (1 dc, 200 hosts!) I don't know whether this would be desirable.